omod/src/main/webapp/resources/htmlforms/dm_htn_initial.xml

```diff
@@ -100,56 +94,67 @@
                             <obs conceptId="138405" answerConceptIds="664,138571,1067" answerLabels="HIV -ve,HIV +ve,Unknown" style="radio"
                                  answerSeparator=""/>
                         </td>
-                        <td colspan="3">
+                        <td colspan="2">
                             <label>Enrolled to care:</label>
                             <obs conceptId="159811" answerConceptIds="1065,1066" answerLabels="Yes,No" style="radio"
                                  answerSeparator=""/>                        
                         </td>
                      </tr>
 
                      <tr>
-                        <td colspan="4">
+                        <td colspan="3">
                             <label>Other:</label>
                             <obs conceptId="160632"/>                        
                         </td>
                     </tr>
 
                      <tr>
-                        <td>
+                        <td colspan="3">
                             <label>Patient covered by NHIF:</label>
-                            <obs conceptId="159811" answerConceptIds="1065,1066" answerLabels="Yes,No" style="radio"
+                            <obs conceptId="1917" style="yes_no"
```

Comment:
@mkwatuha what values do this store in the database. We need a coded value for this option. Just use 1065 and 1066 as answers. OpenMRS datamodel do not support Boolean values